### DISCUSSION TITLE
Is it possible to change the interval in which apalis checks for new jobs

### DISCUSSION THREAD

**Discussion by IgnisDa** [apalis-dev/apalis#173] [Q&A] [answered]
When I set `RUST_LOG=sqlx=trace`, I see that `SELECT * FROM Jobs WHERE (status = 'Pending' OR status = 'Failed' and attempts < max_attempts)) and run_at < $1 AND job_type > $2` is run > 10 times per second. I would like to reduce to once every minute.

How can I do this?

**Comment by geofmureithi** [comment, marked as answer]
You should be able to set this using `fetch_interval`:
https://github.com/geofmureithi/apalis/blob/42b1ab297c5b123c6b1c32a462691937c4939094/examples/postgres/src/main.rs#L44-L50
